### PR TITLE
feat: create preview command

### PR DESCRIPTION
**Pull request by augustoccesar**
co-author: @ostenbom 

### Description
This PR adds a command to linkup-cli and an endpoint to linkup worker so that a whole "persisted" session can be created.

### CODE REVIEW

File: linkup-cli/src/background_booting.rs [hunk 1]
```diff
@@ -91,27 +86,11 @@ pub fn load_config(
         cache_routes: config.cache_routes,
     };
 
-    let update_req_json = serde_json::to_string(&session_update_req)
+    let content = WorkerClient::new(url)
+        .linkup(&session_update_req)
```
Comment (ostenbom) [inline linkup-cli/src/background_booting.rs:90]: Nice!

File: linkup-cli/src/main.rs [hunk 2]
```diff
@@ -185,6 +189,27 @@ enum Commands {
...
+            short,
+            long,
+            value_name = "CONFIG",
+            help = "Path to config file, overriding environment variable."
+        )]
+        config: Option<String>,
+
+        #[arg(
+            help = "<service>=<url> pairs to preview.",
+            value_parser = ValueParser::new(preview::parse_services_tuple),
```
Comment (ostenbom) [inline linkup-cli/src/main.rs:204]: Cool 👍

**Review by ostenbom — APPROVED** [review]
I think we might have some compatibility issues with local-dns, but we can fix them later.

**Comment by augustoccesar** [comment]
> I think we might have some compatibility issues with local-dns

Oh, which compatibility issues do you see happening? 🤔

**Pull request merged**